.github/workflows/error-deltas-watchdog.yaml

```diff
@@ -5,12 +5,14 @@ on:
   schedule:
     - cron: '0 0 * * 3' # Every Wednesday
 
+permissions:
+  contents: read
+
 jobs:
   check-for-recent:
     runs-on: ubuntu-latest
     if: github.repository == 'microsoft/TypeScript'
     permissions:
-      contents: read # Apparently required to create issues

```

Comment:
Can you leave this here, even if it's redundant? We'll likely change the repo root setting too, and then drop all of the top-level ones afterward.